conda/recipes/versions.yaml

```diff
@@ -91,6 +91,8 @@ panel_version:
   - '>=0.9.*'
 pydeck_version:
   - '>=0.3'
+python_confluent_kafka_version:
+  - '=1.3.0'
```

Comment:
Do we want to hard pin to this version or just say `>=1.3.0`?

Reply:
there are some broker changes in versions above 1.3.0 that would cause breaking changes for our current version of librdkafka. Probably needs to be pinned as we would need to make a few code changes if we used a version above 1.3.0

Reply:
We're already pinning `librdkafka =1.4.2`